### PR TITLE
Database/add.test

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: src/test/java/com/fundation/webservice/database/DBConnectionTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @Test
+    public void getConnection_verifyNotNullConnection() {
+        assertNotNull(DBConnection.getInstance().getConnection());
+    }
+
+    @Test
+    public void getInstance_verifyNotNullInstance() {
+        assertNotNull(DBConnection.getInstance());
+    }
+}
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/database/DBConnectionTest.java]: Please add blank line at the end of the source file.

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 2]
```diff
@@ -120,4 +120,13 @@ public void showContent_verifyDateInDB() {
         String expected = infContent.get(2);
```
Comment (LimbertVargas) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: please, change the magic number 2 in a variable

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 3]
```diff
@@ -120,4 +120,13 @@ public void showContent_verifyDateInDB() {
...
         assertEquals(expected, actual);
     }
+
+    @Test
+    public void showContent_verifyRecordPath() {
+        List<String> infContent = new ArrayList<String>();
+        infContent = query.showContent("test1");
+        String actual = "//testPath/path/file.mp4";
+        String expected = infContent.get(3);
```
Comment (LimbertVargas) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: please, change the magic number 3 in a variable.

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 4]
```diff
@@ -49,4 +49,13 @@ public void insertChecksum_verifyInsertChecksumInDB() {
         String expected = infContent.get(0);
         assertEquals(expected, actual);
     }
+
+    @Test
+    public void insertChecksum_verifyDayRecordInDB() {
+        List<String> infContent = new ArrayList<String>();
+        infContent = query.showContent("test1");
+        String actual = "1";
+        String expected = infContent.get(1);
```
Comment (LimbertVargas) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: please change the magic number 1 with a variable name

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+ *
+ */
+
+package com.fundation.webservice.database;
+
+import com.fundation.webservice.common.Util;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: *?
Please be more specific about your imports.

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+import static org.junit.Assert.*;
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: *?
Please be more specific about your imports.

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 7]
```diff
@@ -87,6 +87,7 @@ public List showContent(String checksum) {
         try {
             Connection connection = DBConnection.getInstance().getConnection();
             PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setString(1, checksum);
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java:90]: please change the magic number 1 with a variable name

File: src/test/java/com/fundation/webservice/database/DBConnectionTest.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ *
+ */
+
+package com.fundation.webservice.database;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/database/DBConnectionTest.java]: *?
Please be more specific about your imports.

File: src/test/java/com/fundation/webservice/database/QueryTest.java [hunk 9]
```diff
@@ -49,4 +49,13 @@ public void insertChecksum_verifyInsertChecksumInDB() {
         String expected = infContent.get(0);
```
Comment (LimbertVargas) [inline src/test/java/com/fundation/webservice/database/QueryTest.java]: please change the magic number 0 with a variable name

File: src/main/java/com/fundation/webservice/database/DBConnection.java [hunk 10]
```diff
@@ -63,7 +63,6 @@ private static void initConnection() {
             USER_PASSWORD = Util.getInstance().getConfig().getPassword();
             PORT_CONNECTION = Util.getInstance().getConfig().getPort();
             HOST_NAME = Util.getInstance().getConfig().getHost();
-            System.out.println(WEBSERVER_DB);
             conn = DriverManager.getConnection("jdbc:mysql://" + HOST_NAME + ":" + PORT_CONNECTION + "/" + WEBSERVER_DB, USER_NAME, USER_PASSWORD);
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/DBConnection.java:66]: please change "conn" variable name with a more descriptive name. e.g.: "connection"

File: src/main/java/com/fundation/webservice/database/Query.java [hunk 11]
```diff
@@ -14,7 +14,8 @@
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Class query allows you to perform methods of querying
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/database/Query.java]: add a dot "." in your final name

File: src/test/java/com/fundation/webservice/database/DBConnectionTest.java (hunk 1 above)
Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/database/DBConnectionTest.java]: please add a dot "." in your final name

File: src/test/java/com/fundation/webservice/database/DBConnectionTest.java [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+/**
+ * Class that allows to carry out the connection test with the database.
+ *
+ * @author Jesus Menacho
```
Comment (LimbertVargas) [inline src/test/java/com/fundation/webservice/database/DBConnectionTest.java]: please add a dot "." in your final name

**Pull request merged**